src/main/scala/timeEdit/TimeEdit.scala

```diff
@@ -0,0 +1,173 @@
+package kaptenallocweb.timeEdit
+
+import org.scalajs.dom.{XMLHttpRequest}
+import kaptenallocweb.tabby.Grid
+
+object TimeEdit:
+
+    def fetchData(
+        onLoad: (xhr: XMLHttpRequest) => Unit,
+        onError: (xhr: XMLHttpRequest) => Unit,
+        uri: String =
+          "https://cloud.timeedit.net/lu/web/lth1/ri19566250000YQQ28Z0507007y9Y4763gQ0g5X6Y65ZQ176.csv"
```

Comment:
This url string should perhaps be provided as param to fetchData and the constant placed i main as it needs to be changed every year and should thus not be hard-coded here perhaps?. Makes it easier to find each year. README should be updated with this information on how/where to update TimeEdit url.